crates/storage/db/src/implementation/mdbx/mod.rs

```diff
@@ -445,13 +445,13 @@ impl DatabaseEnv {
     }
 
     /// Creates all the defined tables, if necessary.
-    pub fn create_tables(&self) -> Result<(), DatabaseError> {
+    pub fn create_tables<TS: TableSet>(&self) -> Result<(), DatabaseError> {
```

Comment:
this should be a new function and the `create_tables` fn calls this with the builtin tables, this way we don't have to update any other code